server/testsupport/curiousAssessmentsTestDataBuilder.ts

```diff
@@ -0,0 +1,469 @@
+import type {
```

Comment:
This is a beast of a test data builder file!
It defines several builder functions (for each type) and exports them (at the bottom of the file)

The key bits to note here are that for every type built by one of these functions, they come with sensible default values, but the values are all changeable via the options property for each function.

I'll point out other relevant things on the specific lines 👇